AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift

```diff
@@ -18,23 +16,21 @@ import Foundation
 
 struct ApiConfig {
     
-    static let baseUrl =                       "https://example.com"
-    static let proxyUrl =                       "https://example.com"
+    static let baseUrl =                       "https://api.amahi.org"
+    static let proxyUrl =                      "https://pfe.amahi.org"
+    
+    private static let CLIENT_ID =              "0714493ef3f647209dfe0574fbc3b426220714aaedfd6c007cd54f33e4a0d4bb"
```

Comment:
This file should not be out here. Please ensure you check this before creating PRs.

Reply:
@codedentwickler My apologies, it will never happen again. Is it all okay now then?

Reply:
I fixed it